flask_static_digest/cli.py

```diff
@@ -17,16 +17,27 @@ def digest():
 @with_appcontext
 def compile():
     """Generate optimized static files and a cache manifest."""
-    _compile(current_app.static_folder,
-             current_app.static_folder,
-             current_app.config.get("FLASK_STATIC_DIGEST_BLACKLIST_FILTER"),
-             current_app.config.get("FLASK_STATIC_DIGEST_GZIP_FILES"))
+    for scaffold in [current_app, *current_app.blueprints.values()]:
+
+        if not scaffold.static_folder:
+            continue
+
+        _compile(scaffold.static_folder,
+                 scaffold.static_folder,
+                 current_app.config.get(
+                     "FLASK_STATIC_DIGEST_BLACKLIST_FILTER"),
+                 current_app.config.get("FLASK_STATIC_DIGEST_GZIP_FILES"))
 
 
 @digest.command()
 @with_appcontext
 def clean():
     """Remove generated static files and cache manifest."""
-    _clean(current_app.static_folder,
-           current_app.config.get("FLASK_STATIC_DIGEST_BLACKLIST_FILTER"),
-           current_app.config.get("FLASK_STATIC_DIGEST_GZIP_FILES"))
+    for scaffold in [current_app, *current_app.blueprints.values()]:
+
```

Comment:
Can you remove this extra new line?

Reply:
[af8b1e3 fixed spacing in cli, removed unused imports](https://github.com/nickjj/flask-static-digest/pull/33/commits/af8b1e3de170227097090ef08ea0bb0a875d5b31)

Reply:
Given the other proposed changes would it make sense to rename `for scaffold in` to `for blueprint in`?

Reply:
[0501677 renaming scaffold and ingestScaffold, reworked load_manifest](https://github.com/nickjj/flask-static-digest/pull/33/commits/05016779a5593dd065231c662fd6040c2fb6e09e)